hadoop-cloud-storage-project/hadoop-tos/src/main/java/org/apache/hadoop/fs/tosfs/util/Iterables.java

```diff
@@ -19,7 +19,7 @@
 package org.apache.hadoop.fs.tosfs.util;
 
 import org.apache.hadoop.util.Preconditions;
-import org.checkerframework.checker.nullness.qual.Nullable;
+import org.apache.hadoop.thirdparty.org.checkerframework.checker.nullness.qual.Nullable;
```

Comment:
I suspect this makes the `Nullable` useless, I don't think the static analyzer tools can recognize such a relocated annotation.

Reply:
I think what you said makes some sense, but there are similar users in AzureBFS as well.

https://github.com/apache/hadoop/blob/1566613c725979d0ccda45822dfa275cbd97467a/hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsLease.java#L38